tv_apps/smart-home/test/unit/mock_key_navigation_adapter.js

```diff
@@ -0,0 +1,14 @@
+/* globals evt */
+
+(function(exports) {
+  'use strict';
+
+  var MockKeyNavigationAdapter = function() {
+  };
+
+  MockKeyNavigationAdapter.prototype = evt({
+    init: function() {}
+  });
+
+  exports.MockKeyNavigationAdapter = MockKeyNavigationAdapter;
+})(window);
```

Comment:
We already have a `mock_key_navigation_adapter.js` in `shared/test/unit/mocks/smart-screen/`. I think we should consolidate these two `mock_key_navigation_adapter.js`. And since other apps need `mock_key_navigation_adapter.js` to run tests, I think it is better to place it under shared/